sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSubForSocketIO/api/Microsoft.Azure.WebJobs.Extensions.WebPubSubForSocketIO.netstandard2.0.cs

```diff
@@ -0,0 +1,211 @@
+    public abstract partial class SocketIOAction
+    {
+        protected SocketIOAction() { }
+        public static Microsoft.Azure.WebJobs.Extensions.WebPubSubForSocketIO.AddSocketToRoomAction CreateAddSocketToRoomAction(string @namespace, string socketId, string room) { throw null; }
+        public static Microsoft.Azure.WebJobs.Extensions.WebPubSubForSocketIO.DisconnectSocketsAction CreateDisconnectSocketsAction(string @namespace, System.Collections.Generic.IEnumerable<string> rooms) { throw null; }
+        public static Microsoft.Azure.WebJobs.Extensions.WebPubSubForSocketIO.RemoveSocketFromRoomAction CreateRemoveSocketFromRoomAction(string @namespace, string socketId, string room) { throw null; }
+        public static Microsoft.Azure.WebJobs.Extensions.WebPubSubForSocketIO.SendToNamespaceAction CreateSendToNamespaceAction(string @namespace, string eventName, System.Collections.Generic.IEnumerable<object> parameters, System.Collections.Generic.IList<string> exceptRooms = null) { throw null; }
+        public static Microsoft.Azure.WebJobs.Extensions.WebPubSubForSocketIO.SendToRoomsAction CreateSendToRoomsAction(string @namespace, System.Collections.Generic.IEnumerable<string> rooms, string eventName, System.Collections.Generic.IEnumerable<object> parameters, System.Collections.Generic.IList<string> exceptRooms = null) { throw null; }
+        public static Microsoft.Azure.WebJobs.Extensions.WebPubSubForSocketIO.SendToSocketAction CreateSendToSocketAction(string @namespace, string socketId, string eventName, System.Collections.Generic.IEnumerable<object> parameters) { throw null; }
+    }
+    [Microsoft.Azure.WebJobs.Description.BindingAttribute]
+    [System.AttributeUsageAttribute(System.AttributeTargets.Parameter | System.AttributeTargets.ReturnValue)]
+    public partial class SocketIOAttribute : System.Attribute
+    {
+        public SocketIOAttribute() { }
+        public string Connection { get { throw null; } set { } }
+        [Microsoft.Azure.WebJobs.Description.AutoResolveAttribute]
+        public string Hub { get { throw null; } set { } }
+    }
+    public partial class SocketIOFunctionsOptions : Microsoft.Azure.WebJobs.Hosting.IOptionsFormatter
+    {
+        public SocketIOFunctionsOptions() { }
+        public string ConnectionString { get { throw null; } set { } }
+        public string Hub { get { throw null; } set { } }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        string Microsoft.Azure.WebJobs.Hosting.IOptionsFormatter.Format() { throw null; }
+    }
+    [Microsoft.Azure.WebJobs.Description.BindingAttribute]
+    [System.AttributeUsageAttribute(System.AttributeTargets.Parameter | System.AttributeTargets.ReturnValue)]
+    public partial class SocketIONegotiationAttribute : System.Attribute
+    {
+        public SocketIONegotiationAttribute(string hub) { }
```

Comment:
Shall we have a constructure to support both parameter or a parameterless one, so customer can set `Connection` in code?